4-projects/modules/infra_pipelines/main.tf

```diff
@@ -167,6 +174,17 @@ resource "null_resource" "cloudbuild_terraform_builder" {
   ]
 }
 
+/***********************************************
```

Comment:
Shouldn't we also update the null resource builder so it is only triggered is use_custom_image is false

Reply:
Sorry for delay @bharathkkb · I've added a count to update the null resource builder.